### ISSUE TITLE
datetimes are read with time zone offset

### ISSUE THREAD

**Issue by barrybecker4** [databricks/spark-csv#296]
I'm not sure if this is a question or a bug yet, but let me explain my problem.
When I specify a dateFormat like "MM/dd/yyyy HH:mm:ss" and read in a date like
 "07/09/2015 00:00:00"
It interprets it as having the local (PST) timezone. I really want it to have UTC time zone.
If I print the date with UTC timezone, it report something like 2015-07-09 00:00:00-08:00.
How can I read a date and have it interpreted as UTC?
One way might be to change the date format in the file so that dates end with Z or -00:00 and use a corresponding dateFormat option. But what if I cannot change my datafile date format? Can we have a way to specify the timezone used when reading dates (if it is not specified)? Another timeZone option would allow for this, but I'm not sure if its the best solution.

**Comment by falaki** [comment]
In next version of the package you will be able to specify format. See this PR: https://github.com/databricks/spark-csv/pull/280

**Comment by barrybecker4** [comment]
Yes. I am aware that the next version will allow specifying the format. In fact, we are already using that using a SNAPSHOT. This question/request is about specifying the time zone when reading, instead of using the local time zone. In my case I want to use GMT/UTC time zone. Can we have an additional option for that?

**Comment by falaki** [comment]
If your data has timezone, then you can use format to read them correctly. If they don't I think timezone transformations are better handled with DataFrame transformations (e.g., adding a new column with fixed date).

**Comment by barrybecker4** [comment]
I could add a DataFrame transformation to shift the timezone after the data has been loaded, but I  thought it would be more convenient and efficient to specify it when loading.
